packages/core/src/external/carequality/dq/process-inbound-dq.ts

```diff
@@ -0,0 +1,28 @@
+import { InboundDocumentQueryReq, InboundDocumentQueryResp } from "@metriport/ihe-gateway-sdk";
+import { constructDQErrorResponse, IHEGatewayError, XDSRegistryError } from "../error";
+import { findDocumentReferences } from "./find-document-reference";
+
+export async function processInboundDocumentQuery(
+  payload: InboundDocumentQueryReq
+): Promise<InboundDocumentQueryResp> {
+  try {
+    const documentContents = await findDocumentReferences(payload);
+
+    const response: InboundDocumentQueryResp = {
+      id: payload.id,
+      timestamp: payload.timestamp,
+      responseTimestamp: new Date().toISOString(),
+      extrinsicObjectXmls: documentContents,
```

Comment:
nit: they're just DocRefs in XML format, right? Should we just call it ` docRefXmls` or something?

`extrinsicObjectXmls` is a bit vague

Reply:
This would require a change on IHE GW as well. I'll ticket a review of the "protocol" between API <> IHE GW and add an item to this, ok?

Reply:
https://github.com/metriport/metriport-internal/issues/1545